ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcConfiguration.java

```diff
@@ -43,6 +43,7 @@
   private List<String> hostsAllowlist = Arrays.asList("localhost", "127.0.0.1");
   private boolean authenticationEnabled = false;
   private String authenticationCredentialsFile;
+  private String authenticationAlgorithm = null;
```

Comment:
can we use the JwtAlgorithm type here rather than String, and set it to the default

Reply:
86895b5